src/managers/GuildMemberManager.js

```diff
@@ -135,6 +136,48 @@ class GuildMemberManager extends BaseManager {
     return this._fetchMany(options);
   }
 
+  /**
+   * Edits a member of the guild.
+   * <warn> The user must be a member of the guild </warn>
+   * @param {UserResolvable} user The member to edit
+   * @param {GuildMemberEditData} data The data to edit the member with
+   * @param {string} [reason] Reason for editing this user
+   * @returns {Promise<void>}
+   */
+  async edit(user, data, reason) {
```

Comment:
would it make more sense to name this parameter 'member'? since a member is being edited, the same for the other methods

Reply:
I considered this, but the param being a `UserResolvable` felt awkward with it named `member` so I wasn't sure on that

Reply:
I can see reasons for user but imo member is better, because it is acting on the member, even though it's being found from its user id/object; why can't this be a GuildMemberResolvable though?

Reply:
The GuildMemberResolvable resolution requires the ID to exist in cache, which breaks the uncached support this method is going for.